invokeai/frontend/web/src/features/gallery/components/Boards/BoardsList/GalleryBoard.tsx

```diff
@@ -77,12 +77,12 @@ const GalleryBoard = ({
   const { data: imagesTotal } = useGetBoardImagesTotalQuery(board.board_id);
   const { data: assetsTotal } = useGetBoardAssetsTotalQuery(board.board_id);
   const tooltip = useMemo(() => {
-    if (!imagesTotal || !assetsTotal) {
+    if (!imagesTotal?.total || !assetsTotal?.total) {
```

Comment:
I can't figure out why but the tooltip is not working for me on boards - only the no board tooltip appears (and my list is definitely fully loaded because I'm switching back and forth)

Reply:
good ol' zero is false, gets me every time